src/constants/applications.ts

```diff
@@ -46,3 +46,23 @@ export const expectedSalary = [
   { id: 2, name: 'From 100,000 to 150,000' },
   { id: 3, name: 'From 150,000 to 200,000' },
 ]
+
+export const province = [
+  { id: 1, name: 'Aragatsotn' },
+  { id: 2, name: 'Ararat' },
+  { id: 3, name: 'Armavir' },
+  { id: 4, name: 'Gegharkunik' },
+  { id: 5, name: 'Kotayk' },
+  { id: 6, name: 'Lori' },
+  { id: 7, name: 'Shirak' },
+  { id: 8, name: 'Tavush' },
+  { id: 9, name: 'Vayots Dzor' },
+  { id: 10, name: 'Yerevan' },
+  { id: 11, name: 'Other' },
+]
+
+export const EducationalInstitutionTypes = [
+  { id: 1, name: 'University' },
+  { id: 2, name: 'College' },
+  { id: 3, name: 'Workshop' },
```

Comment:
This one also

Reply:
for this in this moment we don't have api